web/source/kmwinit.js

```diff
@@ -15,18 +15,13 @@
   // Declare KeymanWeb object
   var keymanweb=window['tavultesoft']['keymanweb'];
 
-  if(document.readyState === 'complete')
-  {
-    keymanweb.init(null);
-  }
-  else
-  {
-    var readyStateCheckInterval = window.setInterval(function() {
-      if (document.readyState === "complete") 
-      {
-        window.clearInterval(readyStateCheckInterval);
-        keymanweb.init(null);
-      }
-    }, 10);    
-  }      
```

Comment:
The fix to #108 - we simply don't instantly initialize under any condition.  The user isn't terribly likely to notice a delay of 1/100 of a second.